opencxl/cxl/component/cxl_bridge_component.py

```diff
@@ -44,7 +44,7 @@
 class CxlUpstreamPortComponent(CxlComponent):
     def __init__(
         self,
-        decoder_count: HDM_DECODER_COUNT = HDM_DECODER_COUNT.DECODER_1,
+        decoder_count: HDM_DECODER_COUNT = HDM_DECODER_COUNT.DECODER_20,
```

Comment:
I believe we need more decoders for the USP, or otherwise we don't have enough decoder register space for multiple devices.

Reply:
This is just changing the default value. The actual value assigned during run-time is `HDM_DECODER_COUNT.DECODER_32`.
```
class UpstreamPortDevice(CxlPortDevice):
    def __init__(self, transport_connection: CxlConnection, port_index: int):
        super().__init__(transport_connection, port_index)
        ...
        self._decoder_count = HDM_DECODER_COUNT.DECODER_32
        ...
        self._cxl_component = CxlUpstreamPortComponent(
            decoder_count=self._decoder_count,
            label=label,
        )
```
Why there is a default value to begin with is beyond me, but regardless, this change is not needed.